eox_core/edxapp_wrapper/backends/users_h_v1.py

```diff
@@ -6,14 +6,37 @@
 import logging
 
 from django.db import transaction
+from django.contrib.auth import get_user_model
+from rest_framework.exceptions import APIException
 from openedx.core.djangoapps.lang_pref import LANGUAGE_KEY  # pylint: disable=import-error
 from openedx.core.djangoapps.user_api.accounts.api import check_account_exists  # pylint: disable=import-error
 from openedx.core.djangoapps.user_api.preferences import api as preferences_api  # pylint: disable=import-error
+from openedx.core.djangoapps.user_api.accounts.serializers import UserReadOnlySerializer  # pylint: disable=import-error
 from student.forms import AccountCreationForm  # pylint: disable=import-error
 from student.helpers import do_create_account, create_or_set_user_attribute_created_on_site  # pylint: disable=import-error
 from student.models import create_comments_service_user  # pylint: disable=import-error
 
 LOG = logging.getLogger(__name__)
+User = get_user_model()  # pylint: disable=invalid-name
+
+
+def get_edxapp_user(**kwargs):
```

Comment:
This function is a cornerstone of many features to come. Since it is very important I would like to make sure this function is aware of the tenancy concept (microsite, site_configurations, or others) and that it only retrieves users if the function is being called at the right tenant.

Reply:
Functionality already added.